ci/playbooks/collect-logs.yml

```diff
@@ -67,6 +67,7 @@
               pip3 --version >> ./python.log;
               command -v ansible && ansible --version >> ./python.log;
               pip3 freeze >> ./python.log;
+              dmesg > ./dmesg.log;
```

Comment:
`dmesg -T` to get proper, readable timestamp?

Reply:
@cjeanner fair point!